code/backend/batch/utilities/helpers/config/config_helper.py

```diff
@@ -84,6 +84,8 @@ def get_available_loading_strategies(self):
 
     def get_available_orchestration_strategies(self):
         return [c.value for c in OrchestrationStrategy]
+    def get_available_ai_assistant_types(self):
+        return ["default", "legal assistant"]
```

Comment:
Instead of hard-coding values, use an enum. See OrchestrationStrategy as an example.